Data/ODBC/include/Poco/Data/ODBC/ODBC.h

```diff
@@ -58,6 +58,20 @@
 	#if __has_include(<msodbcsql.h>)
 		#include <msodbcsql.h>
 		#define POCO_DATA_ODBC_HAVE_SQL_SERVER_EXT
+
+		// To enable varchar(max) > 8000 bytes, set to 1.
+		//
+		// Notes:
+		//
+		// - this setting works in conjunction with
+		//   the session "maxFieldSize" property, which
+		//   ultimately determines the max string length.
+		//
+		// - increasing the "maxFieldSize" property may
+		//   affect performance (more memory preallocated
+		//   for prepared statements in order to safely
+		//   accomodate data returned at execution)
+		#define POCO_DATA_SQL_SERVER_BIG_STRINGS 1
```

Comment:
Shall this be configurable in CMake?

Reply:
It could be, but then it should be wrapped in `#ifndef POCO_DATA_SQL_SERVER_BIG_STRINGS` here